src/components/TopBar.js

```diff
@@ -385,7 +394,15 @@ export default _.flow(
             }
           }
         }, ['Confirm'])
-      }, ['Click confirm to remove button forever.'])
+      }, ['Click confirm to remove button forever.']),
+      cookiesModal && h(Modal, {
```

Comment:
This modal should be extracted as a separate component, especially because the same thing appears in two places.